api/requirements-dev.txt

```diff
@@ -32,7 +32,6 @@ et-xmlfile==1.0.1         # via openpyxl
 fancycompleter==0.9.1     # via pdbpp
 flake8==3.7.9             # via pytest-flake8
 gunicorn==20.0.4          # via -r requirements.in
-importlib-metadata==1.5.0  # via pluggy, pytest
```

Comment:
I don't like that dependabot removes dependencies without reason. Out test suite is not comprehensive enough for us to trust these changes. I think we should disable it for the API as it does not add any value.

We should still get security alerts in the security tab https://github.com/code4romania/seismic-risc/security for Django and other supported libraries.